tests/parm/fd_nems.yaml

```diff
@@ -86,6 +86,7 @@
        description: atmosphere export
      #
      - standard_name: Faxa_swnet
+       alias: mean_net_sw_flx
```

Comment:
Bin, will the model still work for other applications if the yaml file is not updated?

Reply:
@junwang-noaa, I believe these changes are backward compatible. Also all the existing RTs (including those cpld and data component coupling ones) went through. But @uturuncoglu and @danrosen25 definitely know more on this than me.

A related question, I thought all the ufs applications use this same fd_nems.yaml file, correct? If that is case, then as long as the application points to this latest develop branch, it will use this update version of fd_nems.yaml file (which will work). And if it points to an older version/commit of ufs-weather-model develop, it will use the corresponding version of the fd_nems.yaml file.

Reply:
Yes, all the ufs applications use this same fd_nems.yaml file. But I need to let workflow group know if they need to update this yaml file when point to this revision of ufs.